mix-otp/docs-tests-and-with.md

```diff
@@ -436,7 +436,7 @@ end
      13:44:10.035 [info]  Application kv exited: :stopped
 ```
 
-С этим простым интеграционным тестом мы можем увидеть, почему интеграционные тесты могут быть медленными. Не только потому, что такой тест не может работать асинхронно, он также предусматривает дорогой установку запуска и старта приложения `:kv`.
+С этим простым интеграционным тестом мы можем увидеть, почему интеграционные тесты могут быть медленными. Не только потому, что такой тест не может работать асинхронно, он также предусматривает дорогой установку запуска и старта приложения `:kv`. На самом деле, ваши тесты могут упасть и привести к таймаутам. Если это так, вы можете настроить вызов :gen_tcp.recv(socket, 0), избавившись от третьего аргумента, который представляет из себя время ожидания в миллисекундах.
```

Comment:
Could you please wrap ":gen_tcp.recv(socket, 0)" into backquotes like other snippets to make it look as `:gen_tcp.recv(socket, 0)`?